app/src/androidTest/java/com/example/p2pchat/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,27 @@
+package com.example.p2pchat;
```

Comment:
Perhaps we should change the package name to something other than "com.example.p2pchat". This could be changed later but it might be easier to do now.